src/cpp/include/openvino/genai/streamer_base.hpp

```diff
@@ -4,16 +4,29 @@
 #pragma once
 
 #include "openvino/genai/tokenizer.hpp"
+#include "openvino/genai/generation_handle.hpp"
+#include <variant>
 
 namespace ov {
 namespace genai {
 
+enum class StreamerRunningStatus {
```

Comment:
```suggestion
enum class StreamingStatus {
```
@Wovchena @sbalandi what do you think?